tests/test_radius_server.rb

```diff
@@ -0,0 +1,182 @@
+    id = '8.8.8.8'
+    id2 = '9.9.9.9'
+    refute_includes(Cisco::RadiusServer.radiusservers, id)
+    refute_includes(Cisco::RadiusServer.radiusservers, id2)
+
+    server = Cisco::RadiusServer.new(id, true)
+    server2 = Cisco::RadiusServer.new(id2, true)
+    assert_includes(Cisco::RadiusServer.radiusservers, id)
+    assert_equal(Cisco::RadiusServer.radiusservers[id], server)
+    assert_includes(Cisco::RadiusServer.radiusservers, id2)
+    assert_equal(Cisco::RadiusServer.radiusservers[id], server2)
+
+    server.accounting_only = true
+    assert_equal(Cisco::RadiusServer.radiusservers[id].accounting_only,
+                 true)
+    assert_equal(server.accounting_only,
+                 true)
+
+    server.authentication_only = true
+    assert_equal(Cisco::RadiusServer.radiusservers[id].authentication_only,
+                 true)
+    assert_equal(server.authentication_only,
+                 true)
+
+    server.acct_port = 44
+    assert_equal(Cisco::RadiusServer.radiusservers[id].acct_port,
+                 44)
+    assert_equal(server.acct_port,
+                 44)
+
+    server.auth_port = 55
+    assert_equal(Cisco::RadiusServer.radiusservers[id].auth_port,
+                 55)
+    assert_equal(server.auth_port,
+                 55)
+
+    server.retransmit_count = 3
+    assert_equal(Cisco::RadiusServer.radiusservers[id].retransmit_count,
+                 3)
+    assert_equal(server.retransmit_count,
+                 3)
+
+    server.key_set = '44444444', nil
+    assert_equal(Cisco::RadiusServer.radiusservers[id].key,
+                 '44444444')
+    assert_equal(server.key,
+                 '44444444')
+
+    server2.accounting_only = true
+    assert_equal(Cisco::RadiusServer.radiusservers[id2].accounting_only,
+                 true)
+    assert_equal(server2.accounting_only,
+                 true)
+
+    server2.authentication_only = true
+    assert_equal(Cisco::RadiusServer.radiusservers[id2].authentication_only,
+                 true)
+    assert_equal(server2.authentication_only,
+                 true)
+
+    server2.acct_port = 44
+    assert_equal(Cisco::RadiusServer.radiusservers[id2].acct_port,
+                 44)
+    assert_equal(server2.acct_port,
+                 44)
+
+    server2.auth_port = 55
+    assert_equal(Cisco::RadiusServer.radiusservers[id2].auth_port,
+                 55)
+    assert_equal(server2.auth_port,
+                 55)
+
+    server2.retransmit_count = 3
+    assert_equal(Cisco::RadiusServer.radiusservers[id2].retransmit_count,
+                 3)
+    assert_equal(server2.retransmit_count,
+                 3)
+
+    server2.key_set = '44444444', nil
+    assert_equal(Cisco::RadiusServer.radiusservers[id2].key,
+                 '44444444')
+    assert_equal(server2.key,
+                 '44444444')
+
+    server.destroy
+    server2.destroy
+    refute_includes(Cisco::RadiusServer.radiusservers, id)
+    refute_includes(Cisco::RadiusServer.radiusservers, id2)
+  end
```

Comment:
Actually, accounting_only, authentication_only and the 2 ports do have default values, so I'll test with those.

Thanks